.tekton/operator-bundle-pipeline.yaml

```
@@ -660,3 +663,130 @@ spec:
       - name: kind
         value: task
       resolver: bundles
+
+  - name: create-acs-style-snapshot
```

I just realized that it would be nice to put a comment saying that this `create-acs-style-snapshot` is the thing which creates a Snapshot representing a valid set of ACS containers unlike the Snapshots created by Konflux automatically.

This could be a comment but it seems that the Tekton spec allows the `description:` attribute here.
```suggestion
  - name: create-acs-style-snapshot
    description: see my comment
```

---

done.